scripts/update_apkid.sh

```diff
@@ -0,0 +1,28 @@
+script_path=$(dirname $0)
+if [ "$script_path" != "scripts" ] && [ "$script_path" != "./scripts" ]; then
+    echo 'Please run script from MobSF directory '
+    echo './scripts/update_apkid.sh '
+    exit  1
+fi
+current_dir=$(pwd)
+rules_dir=${current_dir}/venv/lib/python3.6/site-packages/apkid/rules/
+virtualenv venv -p python3
+source venv/bin/activate
+platform='unknown'
+unamestr=`uname`
+if [[ "$unamestr" == 'Darwin' ]]; then
+  export ARCHFLAGS="-arch x86_64"
```

Comment:
Is openssl required for generating APKiD rules ?

Reply:
will clean that

Reply:
removed